fedora_elections/templates/index.html

```diff
@@ -32,6 +32,10 @@
               <a href="{{ url_for('vote', election_alias=election.alias) }}">
               Vote now!
               </a>
+            {% else %}
+              <a href="{{ url_for('vote', election_alias=election.alias) }}">
+              Revote!
```

Comment:
Maybe more: ``Change your vote``?

Revote sounds like voting again, while it's not a new vote, just an update of the current one